distribution/scenes/textures/patterns/agate.pov

```diff
@@ -2,35 +2,38 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// Agate pattern example
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Agate pattern example agate.pov.
+//
 // -w800 -h600 +a0.3
 
 #version 3.7;
-
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
+#declare Black = srgb <0,0,0>;
+#declare White = srgb <1,1,1>;
```

Comment:
As I'm saying, there's no need to make any such decision in _this_ case.

As for the _general_ question of whether to cut ties with `colors.inc` where necessary to improve the sample scenes, or live with sub-standard sample scenes as long as we have sub-standard include files (which might be the more honest thing to do), I'm torn apart myself. I'd really love to get more feedback from other people on this one before deciding either way.